src/Whoops/Util/TemplateHelper.php

```diff
@@ -57,6 +57,19 @@ public function escapeButPreserveUris($raw)
             "<a href=\"$1\" target=\"_blank\">$1</a>", $escaped
         );
     }
+	
+    /**
+     * Format the given value into a human readable string
+     *
+     * @param  mixed $value
+     * @return string
+     */
+    public function dump($value) {
+        if (class_exists('\Symfony\Component\VarDumper\VarDumper')) {
```

Comment:
infact, it's technically wrong to include it, but it will still work because php trims it under the hood